PhIPSeq_external/Analysis_allergens_and_IEDB/predict_egg_consumption.py

```diff
@@ -0,0 +1,47 @@
+import string
+import seaborn as sns
+from matplotlib import pyplot as plt
+
+from PhIPSeq_external.Analysis_allergens_and_IEDB.prediction_helpers import get_oligos, read_metadata, configure_x_y, \
+    tune_hyper_parameters, plot_correlations, draw_auc, plot_shap
+
+
+def main(tune_params=False):
+    x = get_oligos()
+    meta = read_metadata()
+    meta = meta[['yob', 'gender', 'Eats Egg Recipes']]
+    y = meta['Eats Egg Recipes']
+    meta.drop(columns=['Eats Egg Recipes'], inplace=True)
```

Comment:
the order of these 3 lines is wierd (and why use drop).
Why not simply:
    y = meta['Eats Egg Recipes']
    meta = meta[['yob', 'gender']]

Reply:
In case we change the name of something this is more robust.